control_plane/src/storage.rs

```diff
@@ -235,15 +234,39 @@ impl PageServerNode {
             args.extend(["-c", config_override]);
         }
 
-        let mut cmd = Command::new(self.env.pageserver_bin()?);
-        let mut filled_cmd = fill_rust_env_vars(cmd.args(&args).arg("--daemonize"));
-        filled_cmd = fill_aws_secrets_vars(filled_cmd);
-
-        if !filled_cmd.status()?.success() {
-            bail!(
-                "Pageserver failed to start. See console output and '{}' for details.",
-                datadir.join("pageserver.log").display()
-            );
+        let pageserver_log_path = datadir.join("pageserver.log");
+        let pagesesrver_log_file = fs::OpenOptions::new()
+            .create(true)
+            .write(true)
+            .truncate(true)
```

Comment:
`truncate(true)` is not a good idea. Many tests restart the pageserver, and you don't want to lose the old logs then.